app/controllers/jobs_controller.rb

```diff
@@ -15,6 +15,14 @@ def create
     end
   end
 
+  def new
+    @professions = [["Doctor"], ["Nurse"], ["Assistant Nurse"]]
+    @departments = [["Psychiatric"], ["Surgery"]]
+    @caretypes = [["Hospital"], ["Primarycare"], ["Homecare"], ["Geriatric"], ["Other"]]
```

Comment:
Please consider the snake case format here `@care_types` ?

Reply:
Never thought of that. Will do the same for working_hours